about.css

```diff
@@ -0,0 +1,627 @@
+  padding-top: 30px;
+}
+
+footer .footerMainul {
+  display: flex;
+  flex-direction: row;
+}
+
+.footerMainul .footerFirstul {
+  list-style-type: none;
+  display: flex;
+  padding-left: 25px;
+  flex-direction: column;
+  width: 40%;
+  padding-top: 35px;
+}
+
+.footerMainul .footerSecondul {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  left: 0;
+  flex-direction: column;
+  width: 60%;
+  padding-top: 45px;
+}
+
+.footerFirstul li {
+  font-size: 22px;
+  line-height: 18px;
+  font-weight: bold;
+  color: #000;
+  text-align: left;
+  padding-right: 15px;
+}
+
+.footerSecondul li {
+  font-size: 13px;
+  line-height: 18px;
+  color: #000;
+  padding-left: 0;
+  text-align: left;
+  padding-right: 30px;
+}
+
+/* Menu popup */
+.ham {
+  position: absolute;
+
+  /* keep the hamburger button above everything */
+  z-index: 100;
+  top: 1rem;
+  left: 1rem;
+  width: 3rem;
+  height: 3rem;
+  font-size: 25px;
+}
+
+.xIcon {
+  display: none;
+  z-index: 100;
+  position: fixed;
+}
+
+.menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100vh;
+  color: black;
+  background-color: white;
+  z-index: 1;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  list-style: none;
+  padding-top: 4rem;
+  padding-left: 15px;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
+
+.menu li {
+  padding: 1rem 0;
+}
+
+.menuLink {
+  display: inline;
+  font-size: 20px;
+  color: #ec5242;
+  text-decoration: none;
+}
+
+.menuLink:hover {
+  text-decoration: underline;
+}
+
+.nav-bar .topList {
+  display: none;
+}
+
+.desktopNavFirst {
+  display: none;
+}
+
+.desktopPartner {
+  display: none;
+}
+
+.footerthirdul {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  header img {
+    display: block;
+  }
+
+  .nav-bar {
+    left: 0;
+    display: flex;
+    padding-top: 0;
+    flex-direction: row-reverse;
+    height: 25px !important;
+    background-color: #272a31;
+  }
+
+  .nav-bar .ham {
+    display: none;
+  }
+
+  .nav-bar .menu {
+    display: none;
+  }
+
+  .nav-bar .topList {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    margin: 0;
+    margin-right: 110px;
+  }
+
+  footer {
+    background-color: #272a31;
+  }
+
+  .footerFirstul li {
+    font-size: 22px;
+    line-height: 18px;
+    font-weight: bold;
+    color: white;
+    text-align: left;
+    padding-left: 70px;
+  }
+
+  .footerthirdul li {
+    width: 70%;
+    font-size: 14px;
+    color: white;
+    line-height: 22px;
+  }
+
+  .eventTime li {
+    display: inline-block;
+    line-height: 90px;
+    padding-inline: 0;
+  }
+
+  .desktopNavFirst .secondNavList li {
+    padding-inline: 25px;
+    font-size: 15px;
+    color: #919191;
+  }
+
+  .partner {
+    display: none;
+  }
+
+  /* .partner .desktopPartner li {
+    display: inline;
+    color: #919191;
+    line-height: 60px;
+    padding-inline: 20px;
+    margin-right: 20px;
+  } */
```

Comment:
- [ ] OPTIONAL- Kindly remove this commented out code-block. If you need to refer to previous versions of the code please use Git History.